docs/quickstart-guide.rst

```diff
@@ -157,7 +157,7 @@ If everything has gone well, you should see output from the status command like
 Checking for problems and your first sync
 #########################################
 
-Now that the cobblerd service is up and running, it's time to check for problems. Cobbler's check command will make some
+Now that the Cobblerd service is up and running, it's time to check for problems. Cobbler's check command will make some
```

Comment:
Nitpicking: I would set this as `cobblerd`, in code, without capital C, but YMMV. 😇

Reply:
You are right, this is a mistake.